src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -0,0 +1,99 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "form.njk" import csrfProtection, submitButton %}
+
+{% set title = t('Fees | Money claim') %}
+{% set heading = t('Send your response by email') %}
+
+{% macro newWindow() %}<span class="font-xsmall">{{ t('(opens in a new window)') }}</span>{% endmacro %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="panel-indent">
+        <div class="panel panel-border-wide">
+          <p>{{ t('You must email your response by ') }}
+            <span class="bold">{{ t('%s', { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</span>
```

Comment:
Plase do not use `sprintf`, use new interpolation syntax. Please also use one translation call for the `You must email your response by xxx` paragraph due to reason mentioned above.

Reply:
Interpolation would be along the lines of this. 
```
<span class="bold">{{ t('{{ responseDeadline }}', { responseDeadline: claim.responseDeadline | date } ) }} </span>
```
You can see come examples of it in the privacy-policy.njk file